runtime/src/actor_error.rs

```diff
@@ -111,6 +111,22 @@ macro_rules! actor_error {
     };
 }
 
+/// Convenience macro for generating Actor Errors
+#[macro_export]
+macro_rules! decode_params {
+    // Error with only one stringable expression
+    ($args:ident) => {
+        $args
+            .with_context_code(fvm_shared::error::ExitCode::USR_ILLEGAL_ARGUMENT, || {
+                format!("method expected args")
+            })?
+            .deserialize()
+            .with_context_code(fvm_shared::error::ExitCode::USR_ILLEGAL_ARGUMENT, || {
```

Comment:
I think this should be USR_SERIALIZATION

Reply:
Yes, thank you!